docs/source/development/style-guide/requirement-and-issue-tracking.rst

```diff
@@ -0,0 +1,120 @@
+   :height: 621
+
+
+Overview of Managing Requirements in Github
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* All level 5 requirements will be managed as Github issues.
+* The status of the requirement issue ("Open" / "Closed") is an indication of both development and testing status, so a requirement is not "Closed" until it is implemented and has passed a SIT.
+* Requirement issues should track sub-issues that describe the implementation.
+   * Unlike the L5 requirement issues, these issues close once the code is merged.
+   * Sub-issues allow for better tracking of how SPIKES, implementation issues, and requirements are related to each other.  They also allow the use of L5 requirements as a starting point to see the design history of each feature.
+* If a re-design is proposed that impacts an already tested requirement, that requirement issue should be re-opened and any new SPIKEs or implementation tasks should be appended to the original task list.
+   * Doing this will keep a record of the evolution of implementation decisions that impacted the requirement and a record of the requirement status throughout the project.
+   * This is useful for providing context to new team members or anyone who needs to remember why a particular design was chosen to satisfy a requirement vs another approach.
+* A backlog grooming will take place once per month to re-prioritize and clean up the issue board.
+
+Level 5 Requirements in Github
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Creating level 5 requirements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Level 5 requirements should be created using the "L5 Requirement" issue template.
+   * As part of this template, the L5 summary, Parent (Level 4) Requirement, and list of tasks (sub-issues) are filled in.
+   * If you don't have sub-issues created to link in the task list, you can add text as a reminder or add them later. 
+   * The task list should only be appended to for the lifetime of the project.
+* Upon creation a requirement should be given the following the tags:
+   * "Requirement: Level 5" (you get this for free by selecting the "L5 Requirement" issue template)
+   * The appropriate parent requirement category label (Ex. Parent Req: SDC Data Transfer Monitoring)
+   * "Untested" (This describes the testing status of the requirement)
+* Each requirement should be added to the IMAP project board.
+* Each requirement should be assigned a SIT milestone.
+
+Example:
+
+.. image:: ../../_static/l5-label-example.png
+   :alt: github requirement issue label example
+   :width: 398
+   :height: 448
+
+Github Requirement Issue Lifecycle
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The following diagram gives an example of the general information a level 5 requirement issue should contain as it evolves over the lifecycle of development and testing:
+
+.. image:: ../../_static/git-req-issue-lifecycle.png
+   :alt: Git Requirement issue lifecycle diagram
+   :width: 2161
+   :height: 901
+
+Adding and Tracking sub-issues
```

Comment:
This is a great section. So great, that it might be useful in a different page which can be linked to. For example, in the PR documentation, I would link out to this. Could we move this and the other issue creation sections below into another document (maybe "Github Issue Creation")? Ideally that page would include other standards for issue creation eventually, but for now, you can leave that part off and we can add on later.

Reply:
Yeah, I like this idea!